lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -408,6 +431,10 @@ __e2setcost(2) -- approximation
 local deg2rad = pi / 180
 local rad2deg = 180 / pi
 
+registerFunction("huge", "", "n", function(self, args)
```

Comment:
1. Why call it "huge" and not "infinity" or something?
2. why use registerfunction here, but e2function for the other functions?